app/uk/gov/hmrc/apidocumentation/views/TestUsersDataStatefulBehaviourView.scala.html

```diff
@@ -61,7 +61,9 @@ <h3 class="govuk-heading-s">Test users</h3>
     <p class="govuk-body">Use the same test user’s tax identifiers when testing calls to our APIs. You should also test failure cases by
         using different tax identifiers.
     </p>
-    <p class="govuk-body">You can create as many test users as you like. Each new test user provides a clean test environment.</p>
+    <p class="govuk-body">You can create multiple test users, which will be generated in a default test state.</p>
+    <p class="govuk-body">Test users can be reused, so we recommend checking if you have any unused test users before creating a new one.</p>
```

Comment:
This one sounds odd. How about `we recommend checking if you have any existing test users before creating a new one.`  (`existing` not `unused`)

Reply:
Text is as per ticket... see API-8060 and the google doc mentioned.